Update-AutomationAzureModulesForAccount.ps1

```diff
@@ -109,9 +109,14 @@ function Login-AzureAutomation([bool] $AzModuleOnly) {
     try {
         $RunAsConnection = Get-AutomationConnection -Name "AzureRunAsConnection"
         Write-Output "Logging in to Azure ($AzureEnvironment)..."
-
+        
+        if (!$RunAsConnection.ApplicationId) {
+            $ErrorMessage = "Connection 'AzureRunAsConnection' is incompatible type."

```

Comment:
How about giving a more specific message, perhaps something like `Connection 'AzureRunAsConnection' cannot be used because it does not have the ApplicationId property`?